egret/model_library/unit_commitment/reserve_requirement.py

```diff
@@ -10,12 +10,20 @@
 
 ## system variables and constraints
 from pyomo.environ import *
+from pyomo.core.expr.numeric_expr import LinearExpression
 import math
 
-from .uc_utils import add_model_attr 
+from .uc_utils import add_model_attr, is_var, linear_summation
 from .reserve_vars import check_reserve_requirement
 component_name = 'reserve_requirement'
 
+def _add_reserve_shortfall(model, fixed=False):
```

Comment:
What's the use case for reserve shortfall being fixed/known?

Reply:
When there's no reserve requirement, or it's fixed to 0.

Reply:
Ahh - thanks.